mobile/native/src/dlc_handler.rs

```diff
@@ -175,6 +194,7 @@ impl DlcHandler {
 
                     // TODO(bonomat): we should verify that the proposed amount is acceptable
                     self.node
+                        .inner
                         .accept_dlc_channel_collaborative_close(channel_id)?;
```

Comment:
I guess here we are happy with the original policy of continuing with the channel close after a reconnect.

Reply:
hmm.. good question, I did not look at that. but I think there is no risk of a reserved utxo being used in the meantime as well as a price shift that may impact the originators intend.

To clarify, I think rejecting a pending offer on reconnect is just the first step, as we should as well add a corresponding handling on the coordinator side. Otherwise it would be far to easy for somebody to write a custom client that holds the dlc message for ~1h (3600 seconds), check how the price moved and then either accept or reject the offer.

However, that would require more work as `rust-dlc` currently does not support a cancel flow from accepted. Also its hard to tell if the user tries to game the counterparty or if its just a technical issue.